### PR TITLE
Create 2023 - January 17th.md

### PR DESCRIPTION
**Pull request by vincent6767**
Signed-off-by: Vincent <vincentbiasa6767@gmail.com>

### CODE REVIEW

**Comment by vincent6767** [comment]
@tonybalandiuk Please review Action Item (3) to make sure it's the correct action item.

**Review by tonybalandiuk — COMMENTED** [review]
left a comment - I think we should have a discussion about o3de extras, then the specific issue opened by colin later.

File: meetings/notes/2023 - January 17th.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+7. @AMZN-daimini Streamlining the release notes process updates. 
+   1. Danilo doesn't the have capacity to drive this. Need someone to take over.
+   1. @tonybalandiuk will hold a discussion @AMZN-daimini and @chanmosq to go through the discussion.
+8. @vincent6767 to follow up with TSC on whether we still need the features grid or not.
+   1. Postponed as right now we don't have any replacement for communicating what is usable and what needs work.
+
+## Action Items
+1. @vincentvincent to work with the SIGs to merge the roadmap issue template, follow up with SIG about the roadmap, and follow up with TSC about their feedback related to the roadmap placement.
+2. @tonybalandiuk will work with @vincentvincent to define the roadmap.
+3. @byrcolin to create a GHI in the SIG release defining the proposal and SIG relaese will help mention relevant party.
```
[the remark this answers is not in the document]
Comment (vincent6767) [reply meetings/notes/2023 - January 17th.md:39]: Okay. I'll update the PR.

**Comment by vincent6767** [comment]
Updated the PR. Please review again! @tonybalandiuk

**Review by tonybalandiuk — APPROVED** [review]
looks good

**Pull request merged**